src/seedu/addressbook/data/person/Name.java

```diff
@@ -37,6 +38,14 @@ public static boolean isValidName(String test) {
     }
 
     /**
+     * Returns true if entered name is similar to this name. Two names similar
+     * if full names are the same. Comparison case insensitive.
+     */
+    public boolean isSimilar(Name other) {
+        return this.fullName.toLowerCase().equals(other.fullName.toLowerCase());
```

Comment:
Ideally in TDD, the method implementation should be simple in the first iteration.
Eg. `return false`.
Test are then subsequently added until the tests fail.
The method is then worked on till the test passes, the process repeats itself.